usr/src/lib/libzfs_core/common/mapfile-vers

```diff
@@ -60,19 +60,21 @@ SYMBOL_VERSION ILLUMOS_0.1 {
 	lzc_channel_program;
 	lzc_channel_program_nosync;
 	lzc_clone;
-	lzc_promote;
 	lzc_create;
+	lzc_destroy;
```

Comment:
I think we need to create a new version (`ILLUMOS_0.4`) and add these to them.

Reply:
You only need a new version if you want to treat this as a public and committed interface. Otherwise we should just have a single private version which was what the original release was meant to be, but wasn't named that based on my memory.

Reply:
Does it mean that libzfs_core API is public now and should be documented?

Reply:
@ahrens, no problem with doing that but I am curious if this is a recent requirement.
I recall that in the past new functions were just added in ILLUMOS_0.1.
But I see that there are 0.2 and 0.3 now.